.mergify.yml

```diff
@@ -7,5 +7,10 @@ pull_request_rules:
       conditions:
           - label!=no-mergify
           - '#approved-reviews-by>=1'
-          - status-success=continuous-integration/travis-ci/pr
```

Comment:
I think you should be able to just make this one line, using:
```
- status-success=Run tests
```

I do not believe you need to specify each job. I saw the linked mergify question, and the maintainer's response can be interpreted different ways.  I think while the api may _return_ all of these names, you can still specify the top-level name.  I am ok with leaving this for now though, we can always experiment (and check the api, I haven't found exactly where to look yet) and simplify it later.

Reply:
I'd love to have this be just one step, it would be clearer, less unwieldy and we wouldn't need to remember to change it with every python version update. I read the maintainer answer as requiring every job name to be exact in the mergify file. I'm not sure about the best way to go about testing this either, suggestions welcome!

Reply:
I compressed it into one line - let's be careful with the next MR and see if that works!

Reply:
Cool - keep in mind we won't really know until we merge the PR with the mergify change (so meta!).  So it is possible we will need a rev or two if this doesn't work, but at least we know the actions are godd, since we can see how they ran on your branch when you pushed the commit(s)